acceptance/suites/tests/010-puppetserver-cli/subcommand/irb.rb

```diff
@@ -0,0 +1,41 @@
+require 'test/unit/assertions'
+require 'puppetserver/acceptance/gem_utils'
+
+test_name "Puppetserver 'irb' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
+  cli = "pe-puppetserver"
```

Comment:
Is this right for PE?  I just installed a recent PE drop and saw that "/opt/puppet/bin/puppetserver" and not "/opt/puppet/bin/pe-puppetserver" was laid down as part of the pe-puppetserver package.